docs/articles/sdk/configure-qtcreator.rst

```diff
@@ -0,0 +1,52 @@
+Configuring QtCreator to use the SDK
+====================================
+
+After the installation of the SDK and sourcing environment, there are couple of
+steps to do for running QtCreator in target environment. This roadmap assumes
+that SDK and QtCreator should be installed and environments are sourced already.
+
+Adding a new kit
+--------------------
+
+To be able to cross-compile via QtCreator, new build kit should be added to
+QtCreator.
+
+* Open QtCreator
+* Open Options form from Tools->Options, then click ``Build & Run`` item on the
+left panel of option window.
+* Select ``Kits`` tab and click ``Add`` button.
+* Name your new kit and choose ``Device type`` as ``GenericLinuxOsType``.
+* Then browse and submit the sysroot. If you don't know sysroot location you can
+discover by typing ``echo ${SDKTARGETSYSROOT}`` to console.
```

Comment:
"find it by runnung ``echo ${SDKTARGETSYSROOT}`` in your terminal."

Reply:
this is kinda contradicts with Jeremiah's comment. For now i am using "you can discover by typing ``echo ${SDKTARGETSYSROOT}`` to console"

should i change it like you mentioned here ?

Reply:
There are two things here, console, which I don't like but migt be ok and the word discover which me and Jeremiah think doesn't fit here. Be consistent and either use learn or find instead of discover in the whole article.